src/client.toit

```diff
@@ -115,7 +115,9 @@ class Client:
   Use `net.open` to obtain an interface.
   */
   constructor .interface_
-      --root_certificates/List=[]:
+      --root_certificates/List=[]
+      --security_store/SecurityStore=InMemorySecurityStore:
```

Comment:
We sometimes call this SecurityStoreInMemory to make it easier to match the prefix.